samples/client/petstore/c/Config.cmake.in

```diff
@@ -0,0 +1,5 @@
+@PACKAGE_INIT@
```

Comment:
If a project has its own "Config.cmake.in", your template file will replace it...

Reply:
e.g. https://github.com/kubernetes-client/c/blob/master/kubernetes/Config.cmake.in

Reply:
It is, probably you need to manually remove it from this PR.